nix_update/utils.py

```diff
@@ -46,3 +46,7 @@ def extract_version(version: str, version_regex: str) -> Optional[str]:
         if group is not None:
             return group
     return None
+
+
+def version_is_stable(version: str) -> bool:
+    return version is not None and "-" not in version and "+" not in version
```

Comment:
I am not so sure if this is a good heuritic. I would rather look for `rc`, `alpha`, `beta`, `nightly`.

Reply:
Can we not rely on semantic versioning then?

https://semver.org/#spec-item-9 and https://semver.org/#spec-item-10 are what I used for this.

Reply:
I just grepped for rc1 and the majority of examples did not followed the semver spec. Most of them skipped the `-`.

Reply:
What if I do both? Leave this and also check for the keywords you listed?

Reply:
I am concerned that `-` is often use for something else. I have not seen `+` being used by anything.

Reply:
Ok. `+` is actually used by some Linux distros for example Debian for their internal versioning. I don't think we should use that.

Reply:
But you can also add `preview` to the list.

Reply:
I also added `m[0-9]` for milestones